src/common/income-calc/test/income-calc.service.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { TestBed, async, getTestBed, fakeAsync, tick } from '@angular/core/testing';
+
+import { IncomeCalcService } from '../income-calc.service';
+import { MathService } from "../..";
+
+
+describe('IncomeCalcService', () => {
+
+  let incomeCalcService: IncomeCalcService;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      providers: [IncomeCalcService, MathService]
+    });
+
+    TestBed.compileComponents();
+    const testBed = getTestBed();
+    incomeCalcService = testBed.get(IncomeCalcService);
+
+    }));
+
+  it('calcPlaceIncome() Day', fakeAsync( () => {
+    let context = '1.00';
```

Comment:
context = '1.00';
response = incomeCalcService.calcPlaceIncome(30, 'DAY' , '1');

And change this in other  IT's